node_modules/nodeos-usersfs/scripts/install

```diff
@@ -126,12 +128,10 @@ case $PLATFORM in
 
     mkdir -p `dirname $OUT_DIR`
 
-    genext2fs -b $((DISK_SIZE*1024)) \
+    $GENEXT2FS -b $((DISK_SIZE*1024)) \
```

Comment:
Could you be able to re-add the ```--block-size``` and ```--creator-os``` lines? Or there are not available on the source code? Maybe Ubuntu is using a branch from somewhere... :-/